### PR TITLE
release(hanoi): Release Manifest

### PR DESCRIPTION
**Pull request by bill-mahoney**
Draft PR during code-freeze to prepare for Hanoi Release (1.3.0)

app-service-configurable (v1.3.0)
docker-edgex-consul (v1.3.0)
docker-edgex-mongo (v1.3.0)
edgex-go (1.3.0)
edgex-ui-go (v1.3.0)

Scheduled release Nov 18th - pending TSC vote.

Signed-off-by: Bill Mahoney <bill.mahoney@intel.com>

### CODE REVIEW

File: release/support-rulesengine.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
```
Comment (ernestojeda) [inline release/support-rulesengine.yaml:1]: I dont think we are release support-rulesengine anymore. The repo has been archived

https://github.com/edgexfoundry/support-rulesengine.git
> This repository has been archived by the owner. It is now read-only.

**Review by ernestojeda — CHANGES_REQUESTED** [review]
see my comment

File: release/support-rulesengine.yaml (hunk 1 above)
Comment (bill-mahoney) [reply release/support-rulesengine.yaml:1]: @jpwhitemn , @lenny-intel  can you confirm?  I think per our discussion we still need `docker-edgex-mongo`, `docker-support-logging-go` and `support support-rulesengine` for Hanoi correct?

Comment (lenny-intel) [reply release/support-rulesengine.yaml:1]: > we still need `docker-edgex-mongo`, `docker-support-logging-go` and `support support-rulesengine` for Hanoi correct?

@bill-mahoney , that is correct.

Comment (lenny-intel) [reply release/support-rulesengine.yaml:1]: We haven't made any changes to support-rules engine, so I assume we could just use the Geneva released image? @jpwhitemn , Thoughts?

Comment (lenny-intel) [reply release/support-rulesengine.yaml:1]: Not that we are going to actual `use` it anywhere, but it is still available.

Comment (bill-mahoney) [reply release/support-rulesengine.yaml]: talked in slack, removed it from PR for Hanoi

**Comment by lenny-intel** [comment]
What about edgex-cli?

**Comment by bill-mahoney** [comment]
Last I heard edgex-cli was going to go out a bit after Hanoi

**Comment by bill-mahoney** [comment]
TSC approved Nov 18th. 
@ernestojeda  can you recheck?

**Review by ernestojeda — APPROVED** [review]

**Pull request merged**